### PR TITLE
Make achievements first question and priorities months mandatory

### PR DESCRIPTION
**Pull request by Danman7**
The “how many months will it take” question in priorities and the first question of achievements are now mandatory. I've also found another Text Input related issue described in #198.

**To Test**
Specific steps to test when reviewing:
1. Go to Overview screen in any draft that has both green and either yellow or red answers.
2. Tap on any green answer.
3. Make sure the first question has a * required mark next to the label above (not the placeholder inside the field.
4. Make sure the save button is disabled
5. Enter some text in that field. The placeholder should be gone and only the label and text should appear. The save button should be enabled.
6. Make sure the data is saved.
7. Tap on any yellow or red answer.
8. Save button is enabled.
9. With 0 months selected, taping Save will instead show a required field error below the Months field.

### CODE REVIEW

File: src/screens/__tests__/AddAchievement.test.js [hunk 1]
```diff
@@ -51,6 +51,7 @@ describe('AddAchievement View', () => {
   describe('functionality', () => {
     it('has correct initial state', () => {
       expect(wrapper.instance().state).toEqual({
+        errorsDetected: [],
```
Comment (IvaKop) [inline src/screens/__tests__/AddAchievement.test.js]: Please add a test that ensures button is disabled when the required text field is empty.
